tests/sqlSyncTest.php

```diff
@@ -44,12 +44,13 @@ public function localSqlSync() {
       'yes' => NULL,
     );
     $this->drush('user-create', array($name), $options + array('password' => 'password', 'mail' => $mail));
+    $this->drush('pm-enable', array('comment'), $options + array('yes' => NULL));
```

Comment:
Can we test some field sanitization? If it helps, you can install the standard testing profile so you get 2 content types automatically (article and page).